www/class/centreonACL.class.php

```diff
@@ -2058,24 +2064,21 @@ public function getServiceServiceGroupAclConf($sg_id, $broker = null, $options =
                 . "AND $db_name_acl.centreon_acl.host_id = host.host_id "
                 . "AND $db_name_acl.centreon_acl.service_id = service.service_id ";
         }
+        /*
+         * Using the centreon_storage database to get the information
+         * where the services_servicegroups table provides "resolved" dependencies
+         * for possible components of the servicegroup which can be:
+         *     - simple services
+         *     - service templates
+         *     - hostgroup services
+         */
         $query = $request['select'] . $request['simpleFields'] . " "
             . "FROM ( "
             . "SELECT " . $request['fields'] . " "
-            . "FROM servicegroup, servicegroup_relation, service, host " . $from_acl . " "
-            . "WHERE servicegroup.sg_id = '" . CentreonDB::escape($sg_id) . "' "
-            . "AND service.service_activate='1' AND host.host_activate='1' "
-            . "AND servicegroup.sg_id = servicegroup_relation.servicegroup_sg_id "
-            . "AND servicegroup_relation.service_service_id = service.service_id "
-            . "AND servicegroup_relation.host_host_id = host.host_id "
-            . $where_acl . " "
-            . "UNION "
-            . "SELECT " . $request['fields'] . " "
-            . "FROM servicegroup, servicegroup_relation, hostgroup_relation, service, host " . $from_acl . " "
-            . "WHERE servicegroup.sg_id = '" . CentreonDB::escape($sg_id) . "' "
-            . "AND servicegroup.sg_id = servicegroup_relation.servicegroup_sg_id "
-            . "AND servicegroup_relation.hostgroup_hg_id = hostgroup_relation.hostgroup_hg_id "
-            . "AND hostgroup_relation.host_host_id = host.host_id "
-            . "AND servicegroup_relation.service_service_id = service.service_id "
+            . "FROM " .$db_name_acl . ".services_servicegroups, service, host" . $from_acl . " "
+            . "WHERE servicegroup_id = " . CentreonDB::escape($sg_id) . " "
```

Comment:
please could you use camelCase on $sg_id and could you bind it

Reply:
Should be good now :)